packages/icons/README.md

```diff
@@ -83,4 +83,21 @@ export const Icon = ({ name, , size }) => {
 };
 ```
 
+### Next.js with Pages Router
```

Comment:
**question (non-blocking):** Is the app router okay without the setting?

Reply:
As I mentioned in PR description, the App Router works correctly without the setting.

Reply:
I can push branch with demo if you want

Reply:
I mean here should be mentioned that the app router is ok. That is why I asked. So when the developer see it, it know that it will work in the app router without modification.

Reply:
added